src/utils/url.ts

```diff
@@ -69,3 +69,8 @@ export function getPreviousPageUrl(req: Request, basePath: string) {
 
   return relativePreviousPageUrl;
 }
+
+export const isUpdateOrRemoveJourney = (req: Request): boolean => {
```

Comment:
This will return `false` if user **is** on the Update or Remove journey but BEFORE a company has been loaded. There is a function `isRemoveJourney` higher up in this file that does a more thorough check...

So, I don't think this function and implementation should be in this file. Could it move to the controller/service that's calling it, at which point, presumably you know that you're far enough into the journey to be able to safely check just `entity_number`?

Reply:
I would move it.

Reply:
moved it into the validators